consensus/common/src/test/java/org/hyperledger/besu/consensus/common/jsonrpc/methods/bft/BftProposeValidatorVoteTest.java

```diff
@@ -31,22 +31,22 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
-public class IbftProposeValidatorVoteTest {
+public class BftProposeValidatorVoteTest {
   private final VoteProposer voteProposer = mock(VoteProposer.class);
-  private final String IBFT_METHOD = "ibft_proposeValidatorVote";
+  private final String BFT_METHOD = "bft_proposeValidatorVote";
   private final String JSON_RPC_VERSION = "2.0";
-  private IbftProposeValidatorVote method;
+  private BftProposeValidatorVote method;
 
   @Rule public ExpectedException expectedException = ExpectedException.none();
 
   @Before
   public void setup() {
-    method = new IbftProposeValidatorVote(voteProposer);
+    method = new BftProposeValidatorVote(voteProposer);
   }
 
   @Test
   public void returnsCorrectMethodName() {
-    assertThat(method.getName()).isEqualTo(IBFT_METHOD);
```

Comment:
Be good to test that ibft method name is returned when the legacy flag is set for these RPC methods.